packages/@aws-cdk/cfnspec/lib/schema/augmentation.ts

```diff
@@ -6,6 +6,28 @@ export interface ResourceAugmentation {
    * Metric augmentations for this resource type
    */
   metrics?: ResourceMetricAugmentations;
+
+  /**
+   * Overrides for this resource augmentation
+   */
+  overrides?: ResourceOverrides;
+}
+
+export interface ResourceOverrides {
```

Comment:
rename to `AugmentationOptions` and line up with the schema above, specify the @defaults